packages/the-org/src/investmentManager/plugins/autofun-trader/services/execution/buyService.ts

```
@@ -15,11 +15,227 @@ import { WalletService } from '../walletService';
+  recommend_buy_address: string;
+  reason: string;
+  buy_amount: string | number;
+}
+
+interface ConfigAccount {
+  teamWallet: PublicKey;
+  platformSellFee: number;
+  platformBuyFee: number;
+}
+
+interface BuySignalMessage {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'BuySignalMessage'. Consider to delete it or rename it.

'BuySignalMessage' is defined here:


(lint/suspicious/noRedeclare)